ui/src/app/cohort-search/search-bar/search-bar.component.spec.tsx

```diff
@@ -32,7 +32,6 @@ describe('SearchBar', () => {
     serverConfigStore.set({
       config: {
         ...defaultServerConfig,
-        enableDrugWildcardSearch: false,
       },
     });
```

Comment:
Remove the entire `serverConfigStore` reference.

Reply:
done